doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1581,71 +1548,65 @@ Numeral notations
      Both functions passed to :cmd:`Numeral Notation` must be single
      identifiers.
 
+     .. todo PR: in these 2 exns, how does "must be single identifiers" fit with their
+        declaration as qualid??
+
    .. exn:: The reference @ident was not found in the current environment.
 
+     .. todo PR Should refer to @qualid, but that's a duplicate in vernacular_commands
+        (and gets an error, whereas a duplicate within a file doesn't.  go figure)
+        The reference @qualid was not found in the current environment.
```

Comment:
Remove this error message. It is very common and there is no point in putting it over and over in the documentation.

Reply:
It should be documented somewhere, though, no?

Reply:
It is documented in vernacular_commands.